tests/TestCase/Error/DebuggerTest.php

```diff
@@ -601,6 +601,13 @@ public function testTraceExclude(): void
         $this->assertDoesNotMatchRegularExpression('/^Cake\\\Test\\\TestCase\\\Error\\\DebuggerTest..testTraceExclude/m', $result);
     }
 
+    public function testTraceShortPoints(): void
+    {
+        $result = Debugger::trace(['format' => 'shortPoints']);
+        $this->assertIsArray($result);
+        $this->assertEquals('CORE/vendor/phpunit/phpunit/src/Framework/TestCase.php', $result[0]['file']);
```

Comment:
What does CORE stand for?
I would expect that to be vendor/cakephp/cakephp

Wouldnt this be just `ROOT/...` or even `vendor/phpunit/phpunit/src/Framework/TestCase.php`

Reply:
CORE is a long standing 'short path' that goes to cakephp within `vendor`. We could remove that alias but it will impact debugkit and possibly userland usage.

Reply:
Thats my point: Why would it be `CORE/vendor/phpunit/phpunit/src/Framework/TestCase.php` then?
As that would resolve into `vendor/cakephp/cakephp/vendor/phpunit/phpunit/...`

Reply:
I agree with @dereuromark. The path before the `vendor` folder would be ROOT, not CORE.

Reply:
Ah ok. I didn't pick up on CORE being a prefix to vendor directories. I think this is caused by how we have constants configured in the tests suite. Currently the `CORE` alias is used when paths are prefixed with `CAKE_CORE_INCLUDE_PATH` which we set to `ROOT` in `tests/bootstrap.php`.

Reply:
Then it sounds fine here I guess?

Reply:
The slashes in `$result[0]['file']` need to be normalized to `/` or use `DS` in the expected string to fix the failure on Windows.